include/exiv2/metadatum.hpp

```diff
@@ -264,6 +260,8 @@ class EXIV2API Metadatum {
     @brief Assignment operator. Protected so that it can only be used
            by subclasses but not directly.
    */
+  Metadatum() = default;
```

Comment:
Ey why did you move these constructors? Now the Doxygen groups does not make sense anymore. The creators are empty and the default constructor and the copy constructor are under the documentation for the assignment operator.

Reply:
Fixed.